esp/src/eclwatch/TimingTreeMapWidget.js

```diff
@@ -211,7 +211,9 @@ define([
                     this.stdDev = Math.sqrt(variance);
                     for (var i = 0; i < timers.length; ++i) {
                         var prefix = "other";
-                        if (timers[i].Name.indexOf("Graph graph") == 0) {
+                        timers[i].__hpcc_name = timers[i].Name;
+                        if (timers[i].Name.indexOf("Graph graph") === 0) {
```

Comment:
I am only checking if the string start with "Graph graph".  I am not checking if the string contains "Graph" or "graph"